### DISCUSSION TITLE
Create a Twitter hashtag tracker that tracks the most mentioned non-Iranians

### DISCUSSION THREAD

**Discussion by syavash** [WeAreMahsaAmini/Core#3] [Ideas]
Right now, we have tons of tweets with #MahsaAmini, #Mahsa_Amini, #مهساـامینی, and #OpIran. Many of them are mentioning popular celebrities.

It'll be great if we can build a tool that finds the most mentioned people at any time. This will help everyone to see who are the people that are being mentioned the most and can amplify the effect.

**Comment by siamakhaz** [comment]
here you can find a list of top 50 twitter accounts 
[List of most-followed Twitter accounts](https://en.wikipedia.org/wiki/List_of_most-followed_Twitter_accounts#:~:text=Barack%20Obama%2C%20Justin%20Bieber%2C%20Katy,over%20100%20million%20followers%20each)
and here is Instagram 
[List of most-followed Instagram accounts](https://en.wikipedia.org/wiki/List_of_most-followed_Instagram_accounts#:~:text=Most%2Dfollowed%20accounts,with%20over%20347%20million%20followers.)

**Comment by siamakhaz** [comment]
here is CSV file list of top users
[instagram.csv](https://github.com/WeAreMahsaAmini/Core/files/9660756/instagram.csv)
[twitter .csv](https://github.com/WeAreMahsaAmini/Core/files/9660783/twitter.csv)

**Reply by siamakhaz** [reply]
@syavash

**Comment by Sarahbaqeri** [comment]
Python
